snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -97,10 +97,19 @@ public class Snapshot: NSObject {
         print("snapshot: \(name)") // more information about this, check out https://github.com/fastlane/fastlane/tree/master/snapshot
 
         sleep(1) // Waiting for the animation to be finished (kind of)
-        XCUIDevice.sharedDevice().orientation = .Unknown
+
+        #if os(tvOS)
+            XCUIApplication().childrenMatchingType(.Browser).count
```

Comment:
Fetching the browsers' count is enough to trigger a screenshot? Does the same work for iOS when you just fetch subviews? I'm surprised